FirebaseAuth/Sources/Utilities/FIRAuthErrorUtils.m

```diff
@@ -383,8 +383,8 @@
 /** @var kFIRAuthErrorMessageQuotaExceeded
     @brief Message for @c FIRAuthErrorCodeQuotaExceeded error code.
  */
-static NSString *const kFIRAuthErrorMessageQuotaExceeded = @"The phone verification quota for this "
-                                                            "project has been exceeded.";
+static NSString *const kFIRAuthErrorMessageQuotaExceeded = @"The quota for this operation "
```

Comment:
Will the definition of "operation" be clear to those who get the message?

Reply:
It should be clear by checking where the error is raised. While we can distinguish different operations purely based on the error.